sound/soc/sof/sof-priv.h

```diff
@@ -419,7 +337,7 @@ struct snd_sof_dev {
 	struct snd_sof_mailbox stream_box;	/* Stream position update */
 	struct snd_sof_ipc_msg *msg;
 	int ipc_irq;
-	u32 next_comp_id; /* monotonic - reset during S3 */
+	u32 next_comp_id; /* monotonic */
 
```

Comment:
@ranj063 why is this comment changed here? It doesn't look to have anything to do with this commit.

Reply:
fixed.